src/benchmark/file_io_read_micro_benchmark.hpp

```diff
@@ -36,6 +36,7 @@ class FileIOMicroReadBenchmarkFixture : public MicroBenchmarkBasicFixture {
   uint32_t NUMBER_OF_BYTES = uint32_t{0};
   uint32_t NUMBER_OF_ELEMENTS = uint32_t{0};
   std::vector<uint32_t> numbers = std::vector<uint32_t>{};
+  bool threads_ready_to_executed = false;
```

Comment:
what's the reasoning for making this a class variable as opposed to having it as a local variable per execution?

Reply:
Instead of having for each function this line of code I wanted to reduce the lines of code. Does this have any memory disadvantages?